web/templates/inbox_view/inbox_row.hbs

```diff
@@ -59,8 +59,8 @@
                                 <i class="zulip-icon zulip-icon-mute-new recipient_bar_icon" data-tippy-content="{{t 'You have muted this topic.'}}"
                                   role="button" aria-haspopup="true" aria-label="{{t 'You have muted this topic' }}"></i>
                             {{else if (eq visibility_policy all_visibility_policies.INHERIT)}}
-                                <i class="zulip-icon zulip-icon-inherit recipient_bar_icon" data-tippy-content="{{t 'You will get default notifications for this topic.'}}"
-                                  role="button" aria-haspopup="true" aria-label="{{t 'You will get default notifications for this topic' }}"></i>
+                                <i class="zulip-icon zulip-icon-inherit recipient_bar_icon" data-tippy-content="{{t 'Notifications are based on your configuration for'}}"
```

Comment:
Splitting up a string like this doesn't work. 

https://zulip.readthedocs.io/en/latest/translating/internationalization.html talks about this, but basically the word order for other languages may not put the stream name at the end. You should construct the string in one place, probably just `web/templates/change_visibility_policy_button_tooltip.hbs`, and not define a `data-tippy-content` at all here, preferring to just put the entire translateable string in the template.`git grep z-stream-name` will help you.